lib/min_heap.rb

```diff
@@ -14,18 +14,23 @@ def initialize
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O (log n) where n is the number of nodes in the store
+  # Space Complexity: O(1)
```

Comment:
Since `heap_up` is recursive, you have some space complexity here.  This should also be `O(log n)`